test/martha/martha_v3.test.js

```diff
@@ -30,31 +30,41 @@ const {
     dosObjectWithInvalidFields,
     drsObjectWithInvalidFields,
     expectedObjWithMissingFields,
+    jadeAccessUrlMetadataResponse,
 } = require('./_martha_v3_resources.js');
 
 const test = require('ava');
 const sinon = require('sinon');
 const {
     marthaV3Handler: marthaV3,
-    DrsType,
-    determineDrsType,
     generateMetadataUrl,
     generateAccessUrl,
-    getDrsAccessId,
     getHttpsUrlParts,
-    MARTHA_V3_ALL_FIELDS,
     overridePencilsDownSeconds,
+    PROTOCOL_PREFIX_DRS,
 } = require('../../martha/martha_v3');
+
+const {
+    DrsProvider,
+    determineDrsProvider,
+    DrsProviderInstances: DefaultDrsProviderInstances,
+} = require("../../martha/drs_providers");
+
+const {
+    MARTHA_V3_ALL_FIELDS, MARTHA_V3_ACCESS_ID_FIELDS,
+} = require("../../martha/martha_fields");
+
 const apiAdapter = require('../../common/api_adapter');
 const config = require('../../common/config');
 const { delay } = require('../../common/helpers');
 const mask = require('json-mask');
+const {BondProviders} = require("../../common/bond");
 
 const terraAuth = 'bearer abc123';
 
-const mockRequest = (req, requestFields = MARTHA_V3_ALL_FIELDS) => {
+const mockRequest = (req, requestFields = MARTHA_V3_ALL_FIELDS, forceAccessUrl = false) => {
```

Comment:
Another way to do this would be to put the optional parameters an options object as the last parameter:
```
const mockRequest = (req, { fields = MARTHA_V3_ALL_FIELDS, forceAccessUrl = false }) => {
```
Then the caller would look like:
```
    const request = mockRequest(
        { body: { url: drsUri } },
        fields: MARTHA_V3_ACCESS_ID_FIELDS,
        forceAccessUrl: true
    );
```
To be fair,`fields` gets slightly more verbose, but I think `forceAccessUrl: true` is much better than having to have a `const FORCE_ACCESS_URL = true`.

Reply:
I'll have another look at cleaning this up. I don't love this syntax either but I really didn't want to change the other 40-odd invocations of `mockRequest` to clean up this one.